bin/audit-svg-colors.js

```diff
@@ -293,12 +324,19 @@ function printReplacementRules( replacementObjects ) {
 }
 
 function formatReplacementRules( rules ) {
-	return _.sortBy( rules, 'to.name' ).map( ( rule ) => {
-		const valueFrom = rule.from.value.padEnd( 7 );
-		const valueTo = rule.to.value.padEnd( 7 );
-
-		return `${ valueFrom } → ${ valueTo } (${ rule.to.name })`;
-	} );
+	if ( rules && rules.length ) {
+		/*The native sort modifies the array in place, so we use `.concat()`
+    	to copy the array, then sort.*/
```

Comment:
Feel free to remove that comment, especially if you change it to use Array spread like I suggest below.